<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatchmentDialogBase</class>
 <widget class="QDialog" name="CatchmentDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>241</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Catchment GRID Delination</string>
  </property>
  <widget class="QLineEdit" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>15</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>48</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Flow Direction :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenDialog">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLayers">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbFDLayer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>48</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbFALayer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>87</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>86</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Flow Accumulation :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSGLayer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>123</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>123</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Stream :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>161</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Output :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
